build/dev.sh

```diff
@@ -219,4 +219,9 @@ all() {
   fastlex
 }
 
+if [ $# -eq 0 ]; then
+        echo "usage: $0 <function name>"
```

Comment:
Thanks, there a lot of scripts that need something like this, but we might as well do build/dev.sh first since that's the first thing people will run into.

But please follow the existing style and use two spaces everywhere.

Reply:
Roger that, sorry bout this -- I'll happily change that!